src/services/grantee.test.js

```diff
@@ -1,45 +1,84 @@
-import db, { Grantee } from '../models';
-import { allGrantees } from './grantee';
+import db, { Grantee, Grant } from '../models';
+import { allGrantees, granteeByIdAndRegion } from './grantee';
 
 describe('Grantee DB service', () => {
-  afterEach(() => {
-    jest.clearAllMocks();
+  const grantees = [
+    {
+      id: 63,
+      name: 'grantee 1',
+    },
+    {
+      id: 64,
+      name: 'grantee 2',
+    },
+    {
+      id: 65,
+      name: 'grantee 3',
+    },
+  ];
+
+  beforeAll(async () => {
+    await Promise.all([
+      ...grantees.map((g) => Grantee.create(g)),
+      await Grant.create({
+        id: 65,
+        number: '1145543',
+        regionId: 1,
+        granteeId: 65,
+        status: 'Active',
+      }),
+      await Grant.create({
+        id: 64,
+        number: '1145341',
+        regionId: 1,
+        granteeId: 64,
+        status: 'Active',
+      }),
+    ]);
   });
 
   afterAll(async () => {
+    await Grantee.destroy({ where: { id: grantees.map((g) => g.id) } });
+    await Grant.destroy({ where: { id: grantees.map((g) => g.id) } });
```

Comment:
Flip the order of these destroy statements, Grantee can not be destroyed because Grant relies on the FK relationship. This is probably erroring.